build/stage-release-appcenter.yml

```diff
@@ -72,26 +81,28 @@ jobs:
           displayName: "Remove downloaded artifacts"
           condition: always()
           inputs:
-            SourceFolder: $(Pipeline.Workspace)
+            SourceFolder: $(Pipeline.Workspace)/$(artifactName)
+            RemoveSourceFolder: true
             Contents: '**'
 
 - deployment: AppCenter_Android
-  pool: $(windowsPoolName)
```

Comment:
is that parameter still used?

Reply:
It is in the other stages yes! I think the Windows build and the other app center releases do